.github/workflows/llm_bench-python.yml

```diff
@@ -55,6 +55,11 @@ jobs:
       with:
         name: llm.bench_diff
         path: llm.bench_diff.diff
+    - name: Test tiny-random-qwen of pytorch model on Linux
+      run: |
+        export GIT_LFS_SKIP_SMUDGE=0
+        git clone https://huggingface.co/katuni4ka/tiny-random-qwen
```

Comment:
Can we clone just  latest version? It may save a little time.

Reply:
Updated